rpc.go

```diff
@@ -65,11 +74,16 @@ func New(cfg Config) RPC {
 	if cfg.Name == "" {
 		panic("a service name is required")
 	}
+
+	filters := []transport.Filter(cfg.Filters)
+	interceptors := []transport.Interceptor(cfg.Interceptors)
 	return rpc{
-		Name:      cfg.Name,
-		Registry:  transport.NewMapRegistry(cfg.Name),
-		Inbounds:  cfg.Inbounds,
-		Outbounds: cfg.Outbounds,
+		Name:        cfg.Name,
+		Registry:    transport.NewMapRegistry(cfg.Name),
+		Inbounds:    cfg.Inbounds,
+		Outbounds:   cfg.Outbounds,
+		Filter:      transport.ChainFilters(filters...),
+		Interceptor: transport.ChainInterceptors(interceptors...),
```

Comment:
The public `Config` thing still takes a list of each for convenience. They're compiled down to a single Filter/Interceptor internally.